src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((style) => style.split(':'))
+    .reduce((acc, [key, value]) => {
+      if ((key, value)) {
+        const [_key, _value] = [key.trim(), value.trim()];
+
+        acc[_key] = _value;
```

Comment:
[CODE KNOWLEDGE] - Ensure that the keys in the resulting object are in camelCase format if they contain vendor prefixes (e.g., `-webkit-transition` should become `WebkitTransition`). This is a common practice when converting CSS style strings into JavaScript objects.